README.md

```diff
@@ -453,7 +465,8 @@ Running `lerna` without arguments will show all commands/options.
   "bootstrapConfig": {
     "ignore": "component-*"
   },
-  "packages": ["packages/*"]
+  "packages": ["packages/*"],
+  "registry": "https://registry.npmjs.org"
```

Comment:
I'm definitely glad we're setting `publishConfig.registry` in the created package.json, that's key.